x-pack/legacy/plugins/canvas/i18n/README.md

```diff
@@ -0,0 +1,69 @@
+# Canvas and Internationalization (i18n)
+
+Creating i18n strings in Kibana requires use of the `@kbn/i18n` library. The following outlines the strategy for localizing strings in Canvas
```

Comment:
Can we link to the i18n package guidelines here? https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md